rollup.config.js

```diff
@@ -1,17 +1,38 @@
+const path = require('path');
 const resolve = require('rollup-plugin-node-resolve');
 const commonjs = require('rollup-plugin-commonjs');
 const babel = require('rollup-plugin-babel');
 const replace = require('rollup-plugin-replace');
-const { uglify } = require('rollup-plugin-uglify');
+const { terser } = require('rollup-plugin-terser');
 const json = require('rollup-plugin-json');
 const builtins = require('rollup-plugin-node-builtins');
 const globals = require('rollup-plugin-node-globals');
 const flow = require('rollup-plugin-flow');
 const filesize = require('rollup-plugin-filesize');
 const babelOptions = require('./babel.config');
+const pkg = require(path.join(process.env.INIT_CWD, 'package.json'));
```

Comment:
how about using a package like [read-pkg-up](https://github.com/sindresorhus/read-pkg-up)?

Reply:
Thanks 💯. Just when you think Sindre didn't make a package for it.

Reply:
@tdeekens just found it a few days ago in kcd-scripts. 😆

Reply:
Tried, the promise virus spreads. Top-level async/await is not a thing in Node.js yet. Making things a bit odd. I'd stick with this for now but thanks for the suggestion!

Reply:
Wait a minute: `readPkgUp.sync`.

Reply:
Yeah, that's what I was going to suggest 🤷‍♂️

Reply:
[here](https://github.com/kentcdodds/kcd-scripts/blob/8337bbd16fa4f964f688975f3c510d4705c94932/src/utils.js#L10)